model/project.go

```diff
@@ -1552,9 +1552,9 @@ func FetchVersionsAndAssociatedBuilds(project *Project, skip int, numVersions in
 	// fetch all of the builds (with only relevant fields)
 	buildsFromDb, err := build.Find(
 		build.ByVersions(versionIds).
-			WithFields(build.BuildVariantKey, build.TasksKey, build.VersionKey, build.DisplayNameKey))
+			WithFields(build.BuildVariantKey, bsonutil.GetDottedKeyName(build.TasksKey, build.TaskCacheIdKey), build.VersionKey, build.DisplayNameKey))
```

Comment:
nit, since this is a bit more complicated now, a different line per field may help (like how versions are laid out)

Reply:
done